src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -983,6 +983,20 @@ static void _handleStrokeLineJoinAttr(TVG_UNUSED SvgLoaderData* loader, SvgNode*
     node->style->stroke.join = _toLineJoin(value);
 }
 
+static void _handleStrokeMiterlimitAttr(SvgLoaderData* loader, SvgNode* node, const char* value)
+{
+    char* end = nullptr;
+    const float miterlimit = svgUtilStrtof(value, &end);
+
+    //* SVG2WG: A negative value for stroke-miterlimit must be treated as an illegal value.
+    if (miterlimit < 0.0f) {
+        TVGLOG("SVG", "Negative values for stroke-miterlimit (%f <- %f) are ignored.", node->style->stroke.miterlimit, miterlimit);
```

Comment:
If the negative value is outside the SVG specification, 'TVGERR' would be more appropriate.